shell/src/main/java/alluxio/cli/fsadmin/command/report/SummaryCommand.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command.report;
+
+import alluxio.client.block.RetryHandlingBlockMasterClient;
+import alluxio.client.MetaMasterClient;
+import alluxio.client.RetryHandlingMetaMasterClient;
+import alluxio.master.MasterClientConfig;
+import alluxio.wire.BlockMasterInfo;
+import alluxio.wire.MasterInfo;
+
+import java.util.Map;
+
+/**
+ * Print Alluxio cluster summarized information.
+ */
+public class SummaryCommand {
+  /** Print Alluxio cluster summarized information.*/
+  public static void printSummary() {
+    System.out.println("Alluxio Cluster Summary: ");
+    try (MetaMasterClient client =
+        new RetryHandlingMetaMasterClient(MasterClientConfig.defaults())) {
+      MasterInfo masterInfo = client.getMasterInfo();
+      System.out.println("    Master Address: " + masterInfo.getMasterAddress());
+
+      int webPort = masterInfo.getWebPort();
+      if (webPort != 0) { // Alluxio web services are running
+        System.out.println("    Web Port: " + webPort);
+      }
+
+      System.out.println("    Rpc Port: " + masterInfo.getRpcPort());
+      System.out.println("    Started: " + masterInfo.getStartTime());
+      System.out.println("    Uptime: " + masterInfo.getUpTime());
+      System.out.println("    Version: " + masterInfo.getVersion());
+      System.out.println("    Safe Mode: " + masterInfo.isSafeMode());
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
nit: prettier exception message when master is unavailable?. something like `Please check the master status by ... and do ... to restart`

Reply:
Yeh!, this also one common mistake I always make: run the report CLI before starting the Alluxio cluster.
Thanks for the suggestion!

Codes changed to:
    try (MetaMasterClient client =
        new RetryHandlingMetaMasterClient(MasterClientConfig.defaults())) {
....
    } catch (UnavailableException e) {
      e.printStackTrace();
      System.out.println("Please check the Alluxio master status.");
    } catch (Exception e) {
      e.printStackTrace();
    }